test/beast/core/basic_stream.cpp

```diff
@@ -43,12 +43,12 @@ namespace beast {
 #if 0
 template class basic_stream<
     net::ip::tcp,
-    net::executor,
+    net::any_io_executor,
```

Comment:
it looks like we're fixing code that isn't compiled?

Reply:
I suspect Chris did a global search/replace